src/psyclone/psyir/nodes/datanode.py

```diff
@@ -55,3 +56,24 @@ def datatype(self):
         :rtype: :py:class:`psyclone.psyir.symbols.UnresolvedType`
         '''
         return UnresolvedType()
+
+    def is_character(self, unknown_as=None):
+        '''
+        :param unknown_as: Determines behaviour in the case where it cannot be
+            determined whether the DataNode is a character. Defaults to None,
+            in which case an exception is raised.
+        :type unknown_as: bool
+        :returns: True if this DataNode is a character, otherwise False.
+        :rtype: bool
+
```

Comment:
In the docstring we typically add a blank line between the arguments and the return tags, then another blank line and the "raises" tags. It also misses the Optional type (since it accepts None as value). In this case it should be:
```
...
:type unknown_as: Optional[bool]

:returns: True if this DataNode is a character, otherwise False.
:rtype: bool

:raises Exception:  if the intrinsic type cannot be determined.
```

Reply:
Ah thanks. Applied in c31018b60.